spec/reek/ast/sexp_extensions_spec.rb

```diff
@@ -248,11 +248,23 @@
 end
 
 RSpec.describe Reek::AST::SexpExtensions::SendNode do
-  context 'with no parameters' do
-    let(:node) { sexp(:send, nil, :hello) }
+  context 'with no parameters and no receiver' do
```

Comment:
Maybe we should have one context for the case of `new`, and another for the case of `hello`? Now, it is not clear that you're testing `new` here.